docs/gettinginvolved/acceptance-testing.rst

```diff
@@ -223,6 +235,70 @@ The nodes should be configured to allow key based SSH connections as user ``root
 
    admin/run-acceptance-tests --distribution centos-7 --provider managed --config-file config.yml
 
+If you are using the ``managed`` provider with ``AWS`` nodes, you  will need to supply both the private and public IP addresses for each node.
+AWS nodes do not have public IP addresses configured in the operating system; instead Amazon routes public IP traffic using NAT.
+In this case the acceptance tests need a hint in order to map the private IP address reported by the Flocker ``/state/nodes`` API to the public node address.
+E.g. When a test needs to verify that a container on the node is listening on an expected port or to communicate directly with the Docker API on that node.
+The mapping is supplied to the tests in the ``FLOCKER_ACCEPTANCE_HOSTNAME_TO_PUBLIC_ADDRESS`` environment variable.
+
+If you create nodes using ``run-acceptance-tests --runner=aws --keep`` the command will print out the node addresses when it exits. E.g.
+
+.. code-block:: console
+
+   ./admin/run-acceptance-tests \
+     --keep \
+     --distribution=centos-7 \
+     --provider=aws \
+     --dataset-backend=aws \
+     --config-file=$PWD/acceptance.yml \
+     --branch=master \
+     --flocker-version='' \
+     flocker.acceptance.obsolete.test_containers.ContainerAPITests.test_create_container_with_ports
+
+   ...
+
+   flocker.acceptance.obsolete.test_containers
+     ContainerAPITests
+       test_create_container_with_ports ...                                   [OK]
+
+   -------------------------------------------------------------------------------
+   Ran 1 tests in 14.102s
+
+   PASSED (successes=1)
+   --keep specified, not destroying nodes.
+   To run acceptance tests against these nodes, set the following environment variables:
+
+   export FLOCKER_ACCEPTANCE_DEFAULT_VOLUME_SIZE=1073741824;
+   export FLOCKER_ACCEPTANCE_CONTROL_NODE=54.159.119.143;
+   export FLOCKER_ACCEPTANCE_HOSTNAME_TO_PUBLIC_ADDRESS='{"10.230.191.121": "54.158.225.35", "10.69.174.223": "54.159.119.143"}';
+   export FLOCKER_ACCEPTANCE_VOLUME_BACKEND=aws;
+   export FLOCKER_ACCEPTANCE_NUM_AGENT_NODES=2;
+   export FLOCKER_ACCEPTANCE_API_CERTIFICATES_PATH=/tmp/tmpfvb3xV;
+
+In this case you can copy and paste the ``FLOCKER_ACCEPTANCE_HOSTNAME_TO_PUBLIC_ADDRESS`` value directly into the configuration file. E.g.
```

Comment:
Perhaps a good follow-up would be to get `run-acceptance-tests --keep` to dump out an appropriate `managed` stanza?